supported/osg-htc/osg-hosted-ce/values.yaml

```diff
@@ -154,10 +154,11 @@ Persistence:
   LibCondorCeVolume: null
 
 # Options to allow override of the bosco directory from arbitrary git repos
-# Bosco override dirs are expected in the following location in the git repo:
+# Bosco override dirs default to the following location in the git repo:
 #   <RESOURCE NAME>/bosco_override/
 BoscoOverrides:
   Enabled: false
+  BoscoOverrideDir: null
```

Comment:
Could you add a short comment above here to describe what the intended effect is for this value?

Reply:
Both done. Want to give it a once over before I rebase and merge?